app/src/org/commcare/provider/InstanceProvider.java

```diff
@@ -542,10 +542,14 @@ private void linkToSessionFormRecord(Uri instanceUri) {
             // before moving on. We'll catch any errors here and just eat them
             // (since the task will also try the process and fail if it does.
             if (FormRecord.STATUS_COMPLETE.equals(current.getStatus())) {
+                net.sqlcipher.database.SQLiteDatabase db =
```

Comment:
same question

Reply:
Because I'm referencing both `android.database.sqlite.SQLiteDatabase` and `net.sqlcipher.database.SQLiteDatabase` in the same class. I think the easiest way to that is to not have an import statement for either one, and just reference them both by their full path

Reply:
It's fine to keep the import for the common one as long as you keep the fully qualified reference to the sqlcipher DB here. Not sure if that causes Lint issues or anyhting